### PR TITLE
Update Node 16.16.0 LTS

### PR DESCRIPTION
**Pull request by haslinghuis**
Update to latest LTS version.

NWjs 0.66.1 seems to work also on Debian but uses Node 18 which will be released as LTS version around October.

### CODE REVIEW

**Review by McGiverGim — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> PASS
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Comment by meoso** [comment]
> Do you want to test this code? Here you have an automated build: [Betaflight-Blackbox-Explorer-Linux](https://github.com/betaflight/blackbox-log-viewer/suites/7428399048/artifacts/303407695) 

Same problem as Configurator packaging here as well :(  
`dpkg-deb: error: archive 'betaflight-blackbox-explorer_3.7.0_amd64.deb' uses unknown compression for member 'control.tar.zst', giving up`

i wonder if `ubuntu-22.04` is the problem source.

**Comment by meoso** [comment]
portable fails video-export on Debian 11.
```
$ ./betaflight-blackbox-explorer 
libva error: /usr/lib/x86_64-linux-gnu/dri/iHD_drv_video.so init failed
[300672:300672:0720/133717.465158:ERROR:sandbox_linux.cc(379)] InitializeSandbox() called with multiple threads in process gpu-process.
[300674:300681:0720/133718.297032:ERROR:nss_util.cc(286)] After loading Root Certs, loaded==false: NSS error code: -8018
[300674:300681:0720/133718.297322:ERROR:cert_verify_proc_builtin.cc(603)] No net_fetcher for performing AIA chasing.
[300642:300665:0720/133803.329567:ERROR:select_file_dialog_impl_portal.cc(698)] Portal returned error: org.freedesktop.portal.Error.InvalidArgument: invalid filter: name is empty
```
 but portable works in Xubuntu 20.04

**Pull request merged**
